Tests/run_tests.py

```diff
@@ -165,6 +173,12 @@ def is_numpy():
     DOCTEST_MODULES.remove("Bio.SeqIO")
     DOCTEST_MODULES.remove("Bio.SearchIO")
 
+# Skip doctest for all modules outside of 'Bio' package prior to Python 3.3
+if float(sys.version[:3]) < 3.3:
+    for i, name in enumerate(DOCTEST_MODULES):
+        if is_outside_bio(name):
```

Comment:
I don't think the extra function is needed, how about ``if not name.split(".", 1)[0].startswith("Bio"):`` here?